app/Http/Controllers/ConferencesController.php

```diff
@@ -185,8 +188,30 @@ public function destroy($id)
         return redirect('conferences');
     }
 
+    public function dismiss($conferenceId)
```

Comment:
These controller methods would be a good place to use [implicit route model binding](https://mattstauffer.com/blog/implicit-model-binding-in-laravel-5-2/).

It would look something like this:
![image](https://user-images.githubusercontent.com/2329654/54174393-9b053300-445c-11e9-9c49-67ef244fb557.png)

You would also need to change the routes in `web.php` to use `{conference}` instead of `{id}`.

Reply:
This doesn't need to happen for this PR. We could add a new task later to use implicit route model binding.